src/store/__mocks__/storeBuilder.ts

```diff
@@ -164,7 +164,7 @@ export class FakeStoreBuilder {
       this.state.workspaces.workspaces = Object.assign([], options.workspaces);
     }
     if (options.workspacesLogs) {
-      this.state.workspaces.workspacesLogs = Object.assign({}, options.workspacesLogs);
+      this.state.workspaces.workspacesLogs = options.workspacesLogs;
```

Comment:
Shallow copy of a map may be done like this:

```suggestion
      this.state.workspaces.workspacesLogs = new Map(options.workspaceLogs);
```

Reply:
Ok